pgl/utils/log_writer.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" log writer setup
+"""
+import sys
+
+LogWriter = None
+
+if int(sys.version[0]) == 3:
```

Comment:
I

Reply:
It would be better to add more comments here, since it's a little bit tricky.
Is it possible not dump any log if user use python2?